src/frontend/src/pages/PIAForm/index.tsx

```diff
@@ -476,64 +506,35 @@ const PIAFormPage = () => {
     if (!pia?.title) {
       invalid = true;
       formId = 'title';
+      setValidationMessages((prevState) => ({
+        ...prevState,
+        piaTitle: 'Error: Please enter a title.',
+      }));
     }
-    if (!pia?.ministry && !invalid) {
+    if (!pia?.ministry) {
       invalid = true;
       formId = 'ministry-select';
+      setValidationMessages((prevState) => ({
+        ...prevState,
+        piaMinistry: 'Error: Please select a ministry.',
+      }));
     }
-    if (!pia?.branch && !invalid) {
+    if (!pia?.branch) {
       invalid = true;
       formId = 'branch';
+      setValidationMessages((prevState) => ({
+        ...prevState,
+        piaBranch: 'Error: Please enter a branch.',
+      }));
     }
-    if (!pia?.drafterName && !invalid) {
-      invalid = true;
-      formId = 'drafterName';
-    }
-    if (!pia?.drafterTitle && !invalid) {
-      invalid = true;
-      formId = 'drafterTitle';
-    }
-    if ((!pia?.drafterEmail || pia.drafterEmail === undefined) && !invalid) {
-      invalid = true;
-      formId = 'drafterEmail';
-    }
-    if (!pia?.leadName && !invalid) {
-      invalid = true;
-      formId = 'leadName';
-    }
-    if (!pia?.leadTitle && !invalid) {
-      invalid = true;
-      formId = 'leadTitle';
-    }
-    if (!pia?.leadEmail && !invalid) {
-      invalid = true;
-      formId = 'leadEmail';
-    }
-    if (!pia?.mpoName && !invalid) {
-      invalid = true;
-      formId = 'mpoName';
-    }
-    if (!pia?.mpoEmail && !invalid) {
-      invalid = true;
-      formId = 'mpoEmail';
-    }
-    if (!pia?.initiativeDescription && !invalid) {
+
+    if (!pia?.initiativeDescription) {
       invalid = true;
       formId = 'initiativeDescription';
-    }
-    if (!pia?.initiativeScope && !invalid) {
-      invalid = true;
-      formId = 'initiativeScope';
-    }
-    if (!pia?.dataElementsInvolved && !invalid) {
-      invalid = true;
-      formId = 'dataElementsInvolved';
-    }
-    if (pia?.hasAddedPiToDataElements === false) {
```

Comment:
And also validate this if 'no pi' is selected?

Reply:
good catch, just ask UI designer for clarification